build.sbt

```diff
@@ -30,10 +30,12 @@ lazy val pluginSettings = Seq(
     ),
     libraryDependencies ++= Seq(
       "com.lihaoyi"   %% "scalatags" % "0.6.0",
-      "org.scalactic" %% "scalactic" % "3.0.0"
+      "org.scalactic" %% "scalactic" % "3.0.0",
+      "org.scalatest"  %% "scalatest"  % versions("scalatest")  % "test",
+      "org.scalacheck" %% "scalacheck" % versions("scalacheck") % "test"
     ),
     scalafmtConfig in ThisBuild := Some(file(".scalafmt"))
-  ) ++ reformatOnCompileSettings ++ addTestLibs(vAll, "scalatest", "scalacheck")
+  ) ++ reformatOnCompileSettings
```

Comment:
Yes, it's still valid, but actually, I would like to rewrite the current dependency injection from sbt-catalysts-extras plugin. Maybe, it just a personal preference (I'm getting only the part that makes really sense to me).